packages/gestalt/src/Box.js

```diff
@@ -80,96 +54,818 @@ extra runtime typechecks in the transformers for performance.
 
 */
 
+type UnionRefs = HTMLDivElement | HTMLAnchorElement;
+
+// These types are mostly defined in boxTypes.js. Unfortunately to get generated docs
```

Comment:
This makes me sad. ☹️ I already have a ticket to write a Babel plugin that will inline imported types, which I'll get to in the near-ish future. Once that's merged I'll undo these changes, but I didn't want this PR held up until I have the time for that work.